### DISCUSSION TITLE
Exporting/Importing Collections

### DISCUSSION THREAD

**Discussion by irakli2206** [pocketbase/pocketbase#1084] [Q&A] [answered]
I wanted to export and import collections including their data, but it seems that this only exports and imports the collection schema. Is data exporting in the plans or is this how it's going to stay in the future?

**Comment by ganigeorgiev** [comment, marked as answer]
Data import/export is in the roadmap but before that we'll have to implement transactional bulk create/updated/delete. You can subscribe to this issue - https://github.com/pocketbase/pocketbase/issues/48#issuecomment-1211775627
